Source/objects.cpp

```diff
@@ -5220,26 +5206,29 @@ void ObjChangeMapResync(int x1, int y1, int x2, int y2)
 	}
 }
 
-void TryDisarm(int pnum, int i)
+void TryDisarm(Player &player, int i)

```

Comment:
```suggestion
void TryDisarm(const Player &player, int i)
```
(the object reference would need to be non-const)